### PR TITLE
feat: Add @stop1204 as a contributor 

### LINKED ISSUE #456: Feature:Add @stop1204 as a contributor
### Description

As part of the Intro to Open Source course, we are encouraged to contribute to this repo.

### Suggested solution

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by stop1204**
## Description

This PR adds @stop1204 as a contributor.

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Related Issues

Closes #456
### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [ x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [ x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)

image

## [optional] What GIF best describes this PR or how it makes you feel?

🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -2595,8 +2595,31 @@
         "question",
         "bug"
       ]
+    },
+    {
```
Comment (BekahHW) [inline .all-contributorsrc:2599]: It looks like you're adding two people here. You should only be adding one.

**Review by stop1204 — COMMENTED** [review]
done

**Review by stop1204 — COMMENTED** [review]
done

**Review by BekahHW — APPROVED** [review]

**Review by adiati98 — APPROVED** [review]
Well done and thank you for your contribution, @stop1204! ✨ 

If you haven't, you can join our [Community](https://github.com/orgs/open-sauced/discussions/1).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

git merge GIF

**Pull request merged**